### PR TITLE
Update benchmark_config.json

### PR DESCRIPTION
**Pull request by helyho**
change framework name from voovan to voovan-mvc

### CODE REVIEW

File: frameworks/Java/voovan/benchmark_config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "framework": "voovan",
+  "framework": "voovan-mvc",
```
Comment (nbrady-techempower) [inline frameworks/Java/voovan/benchmark_config.json:2]: I'm confused about this name change. If the framework is called `voovan`, we'd prefer this to match the directory `Java/voovan` but you could change the `display_name` attribute to say `voovan-mvc`. If the framework name is changing, then you can change the directory to `voovan-mvc` as well.

Comment (TonyPythoneer) [reply frameworks/Java/voovan/benchmark_config.json:2]: I agree with what @nbrady-techempower talks about.

If the lib calls `voovan` in Java ecosystem, please keep the origin name. Don't confuse the other people who will contribute to this project in the future.

Comment (helyho) [reply frameworks/Java/voovan/benchmark_config.json:2]: ok i will fixed it

**Comment by helyho** [comment]
please cancel this travis job. because this pr is droped. thanks.

**Pull request closed without merging**